dev-requirements.txt

```diff
@@ -2,3 +2,5 @@ pre-commit
 pytest
 pytest-cov
 coverage
+fuzzywuzzy
```

Comment:
Why adding those?
I'd prefer not to duplicate the dependencies already listed in `setup.py`

Reply:
I did not run 'setup.py' to avoid installing the project in my environment, so I just installed the packages in the dev-requirements.txt which was incomplete in that regard.

I do not know what the general rule is about packages that should be listed there.

With 'pre-commit run py.test' it could be argued that this file is no longer needed.

Also, apparently the more common naming would be 'requirements-dev.txt' based on this pypy page: https://pypi.org/project/requirements-dev.txt/ .  There is no protection against a malicious 'dev-requirements.txt' (yet).


This [non-authorative answer](https://stackoverflow.com/questions/43658870/requirements-txt-vs-setup-py#answer-43659126) suggests that 'requirements.txt' helps to setup the development environment and that 'setup.py' helps create packages to redistribute.  
So according to that rationale, the 'requirements-dev.txt' file should contain all packages.

Reply:
Then just include `-e .` in this file, no need to repeat the packages :)

Reply:
I added `-e .` which adds kind of a circular dependency in the ci.yml.

```
ERROR: Cannot install pre-commit-hooks 1.1.13 (from /home/runner/work/pre-commit-hooks/pre-commit-hooks) because these package versions have conflicting dependencies.
```

So I disabled installing the packages from 'requirements-dev.txt' -  pre-commit should install them as needed - the tests now pass.